vdbench.py

```diff
@@ -1,367 +1,367 @@
-# -*- coding: utf-8 -*-
-#pylint : disable=E0401
-"""
-@author: SUSHANT KEDAR				
-DATACORE SOFTWARE PVT LTD CONFIDENTIAL
-THIS SPEC IS THE PROPERTY OF DATACORE SOFTWARE PVT LTD.IT SHALL NOT BE
-COPIED, USED,TRANSLATED OR TRANSFERRED IN WHOLE OR IN PART TO ANY THIRD
-PARTY WITHOUT PRIOR WRITTEN PERMISSION OF DATACORE SOFTWARE PVT LTD.
-File Name	:	vdbench.py
-Description	:	This script used to execute vdbench tool and create
-                HTML reports.
-
-"""
-import os
-import sys
-from configparser import ConfigParser
-import subprocess
-path_ = os.path.abspath("../../../Interface/REST")
-sys.path.insert(0, path_)
-from ILDC import ILDC
-sys.path.insert(0, os.path.abspath("../../../Lib/VdBenchLib"))
-from error_log import LogCreat
-
-class VdBenchRun():
-    '''
-    Class:- VdBenchRun
-    This class executes vdbench 4k fill and VSI/VDI/ORACLE/SQL
-    wokload as user request.
-    Arguments : None
-    Return: None
-    '''
-    def __init__(self):
-        configur = ConfigParser()
-        configur.read(r"../../../Config/VdBench_config/VDBench_config.ini")
-        self.vdbench_path = configur.get('Vdbench run', 'vdbench_executable_path')
-        self.time_stamp = configur.get('first run', 'start')
-        self.build = ''
-        self.new_ = ''
-    def run(self,vd_name, workload, diskindex):
```

Comment:
variable name must have underscore between 2words. disk_index is prefferred

Reply:
Rules for Python variables:

A variable name must start with a letter or the underscore character
A variable name cannot start with a number
A variable name can only contain alpha-numeric characters and underscores (A-z, 0-9, and _ )
Variable names are case-sensitive (age, Age and AGE are three different variables)

So it will work no need to change it